src/main/flight/pid.c

```diff
@@ -56,6 +56,9 @@
 #include "sensors/gyro.h"
 #include "sensors/acceleration.h"
 
+#ifdef USE_RPM_FILTER
```

Comment:
Is there a compilation problem if this isn't conditional? Headers are normally included unconditionally in Betaflight.

Reply:
ok